gen_mod_headers

```diff
@@ -140,4 +145,41 @@ for f in $(find . -name "Kconfig*"); do
 	cp $f "$target_dir/$f"
 done
 
+echo Fixing up script binaries...
+
+push "$target_dir"
+
+# Make some backups so we can restore sanity after we're done.
+cp scripts/basic/Makefile .backup.basic.Makefile
```

Comment:
@agherzan It doesn't actually copy in anything to `xscripts`, xscripts receives ONLY the binaries for the host arch because `make O=...` is invoked, we copy in the source to `scripts` later.